src/swc_util.rs

```diff
@@ -198,3 +198,9 @@ impl AstParser {
   //   }
   // }
 }
+
+impl Default for AstParser {
+  fn default() -> Self {
+    Self::new()
+  }
```

Comment:
Unrelated change, please revert

Reply:
Alright, seems it's required

Reply:
If you want I can open another pull for the sake of keeping changes related

Reply:
It's fine to include it it this PR